modular_citadel/code/modules/reagents/chemistry/reagents.dm

```diff
@@ -0,0 +1,5 @@
+/datum/reagent/proc/on_mob_attacked(mob/living/M) //Handler for reagent effects that occur when the mob with it inside it is attacked
+	return
+
+/datum/reagent/water/on_mob_attacked(mob/living/M)
```

Comment:
ignore this, this is just what I use for testing the proc. It'll be removed once I get things running proper